### ISSUE TITLE
Filtered table export

### ISSUE THREAD

**Issue by maciazek** [Kreyu/data-table-bundle#143]
Hello again,
while duplicated queries discussed in my previous issue are not a big deal for me, I would like to ask you one more question that I find way more important: is there a way to export only filtered rows from a table? I'm not sure if it is a bug or I'm missing something, but this is the only thing I can't figure out how to implement with your tool. I think about it this way: when I have filters set on the table and then execute export, only filtered data should be returned. Is there a way to achieve this result?

Thank you for efforts you put in this project so far! I'm really glad there's finally something way more straightforward to integrate in Symfony app than good old DataTables.net :)

**Comment by Kreyu** [comment]
Hey @maciazek, the filters were always respected in the export, maybe I broke something with the recent changes... I'll take a second look at all of this in the upcoming week, after the vacation. Thanks for the report!

**Comment by Kreyu** [comment]
Fixed in 0.25.1

**Comment by maciazek** [comment]
Yes, I can confirm that now in v0.25.2 export handles filters correctly. Thank you!
